### PR TITLE
[MongoDB] Added ssl mappings in handlebar file for mongodb data streams

### LINKED ISSUE #4669: [MongoDB] Add the SSL support for all the data stream for mongoDB package
This bug-fix issue will be tracking the progress of SSL support for mongoDB package.

- [x] Add the SSL support in handlebar files for all the data stream.
- [x] Manual Testing.

SDH: https://github.com/elastic/sdh-beats/issues/2739

### PR DESCRIPTION
**Pull request by ManojS-shetty**
## What does this PR do?

In this PR we are adding the SSL mappings to the handlebar file in mongodb integration all data streams.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone the PR.
- Run the elastic-package stack up command.
- Add the mongoDb integration.
- In Integration UI provide all host related details.
- Enrol the elastic agent.
- MongoDB Metrics should be flowing to the Elastic-search.

## Related issues

- Closes #4669

### CODE REVIEW

**Comment by ManojS-shetty** [comment]
> @ManojS-shetty : Have we done secure tests to verify these ?

Yes it has been verified against self signed certificate secured mongodb.

File: packages/mongodb/data_stream/status/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -7,6 +7,16 @@ hosts:
 password: {{password}}
 {{/if}}
 period: {{period}}
+ssl.enabled: {{ssl.enabled}}
+ssl.verification_mode: {{ssl.verification_mode}}
+{{#if ssl.certificate_authorities}}
+ssl.certificate_authorities:
+{{#each ssl.certificate_authorities}}
```
Comment (ishleenk17) [inline packages/mongodb/data_stream/status/agent/stream/stream.yml.hbs:14]: There are no default values that we would want to add for these in the manifest file ?

Comment (ManojS-shetty) [reply packages/mongodb/data_stream/status/agent/stream/stream.yml.hbs:14]: No there is no need of default values here as the option will be disabled by default, also i have seen in other integration like kubernetes are not defining any default values for these mapping.

**Review by ishleenk17 — APPROVED** [review]
Looks good !

**Pull request merged**